packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -2,7 +2,10 @@
   <div
     class="analytics-chart-shell"
   >
-    <div class="chart-header">
+    <div
+      v-if="(hasValidChartData && resultSetTruncated && maxEntitiesShown) || chartTitle || (allowCsvExport && hasValidChartData)"
+      class="chart-header"
+    >
```

Comment:
Conditionally show the blow.

Otherwise, we end up with a nasty chunk of white space above the chart. I can simplify this check further, this is temporary.